en/docs/debug-the-code/debugging-configurations.md

```diff
@@ -0,0 +1,96 @@
+| `type`  	| The type of debugger to use for this launch configuration. The attribute value must be kept as `ballerina` for all Ballerina debugging configuration types.                   	|
+| `request` 	| The request type of this launch configuration. Currently, `launch` and `attach` are supported.                                               	|
+
+### Attributes for the `launch` configurations
+
+The following attributes are supported for all the Ballerina `launch` configurations related to the Ballerina code available locally.
+
+| Attribute        	| Description                                                                                                                                                                                                                                                                                                                                                           	|
+|------------------	|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| `programArgs`    	| Any program arguments that are required to be passed into the `main` function of the Ballerina program to be launched can be passed as a list of strings.                                                                                                                                                                                                              	|
+| `commandOptions` 	| If required, you can configure command options for the Ballerina program to be launched as a list of strings. You can see the list of all the available command options by executing the following CLI commands in your terminal. <br/><br/> - For the `Ballerina Debug` configuration: `bal run --help` <br/> - For the `Ballerina test` configuration: `bal test --help` 	|
+| `env`            	| Any environment variables you need to configure for launching the Ballerina program can be passed as a map of strings (name and value).                                                                                                                                                                                                                                	|
+| `debugTests`     	| Indicates whether to debug the tests for the given script.                                                                                                                                                                                                                                                                                                             	|
+
```

Comment:
It seems we have missed the documentation for `terminal` attribute in here. Refer to https://ballerina.io/learn/debug-ballerina-programs/#ballerina-launchjson-attributes 

<img width="778" alt="Screenshot 2023-05-16 at 22 32 25" src="https://github.com/wso2/docs-ballerina-vscode-extension/assets/29032600/aa53b5b0-425a-4457-92da-134e952b161d">

Reply:
Added this.